docs/pipelines/security/resources.md

```diff
@@ -40,7 +40,7 @@ At the organization or project level, you may choose to limit the scope of the A
 When you do this, Azure Pipelines will add two more protections:
 
 * The access token given to the agent for running jobs will only have access to repositories explicitly mentioned in the `resources` section of the pipeline.
-* Repositories added to the pipeline will have to be authorized by someone with read access to the repository the first time that pipeline uses the repository.
+* Repositories added to the pipeline will have to be authorized by someone with contribution access to the repository the first time that pipeline uses the repository.
```

Comment:
```suggestion
* Repositories added to the pipeline will have to be authorized by someone with contributor access to the repository the first time that pipeline uses the repository.
```
I think that it should be contributor access. Could you fix this? It's good to merge otherwise.